ietf/static_src/css/iab.scss

```diff
@@ -171,3 +181,34 @@
 .iab_body {
     background-color: rgba(62, 139, 193, 0.08);
 }
+
+.dropdown-toggle__iab {
+    &:after {
+        display: inline;
+        vertical-align: -0.75rem;
+    }
+}
+
+.iab-tabs > li {
+    font-size: rem(16);
+    line-height: 1;
+}
+
+.iab-tabs > li > a {
+    padding: rem(16) rem(18) rem(10);
+    text-align: left;
+}
+
+.iab-tabs > li > .active {
+    color: var(--color-iab-navy) !important;
+    border-bottom: 2px solid black !important;
+}
+
+.iab-tabs > li > :not(.active) {
+    color: #616466 !important;
```

Comment:
Do we not have a colour variable to use?

Reply:
We didn't, but now we do